### ISSUE TITLE
NTR: suture closure terms

### ISSUE THREAD

**Issue by sbello** [geneontology/go-ontology#23213]
Please provide as much information as you can: 
MP, HPO, and ZP all have phenotypes related to premature and delayed closure of sutures. The UPheno group would like to use GO process terms to create patterns for these terms.
See links for existing MP and HPO phenotype terms
http://www.informatics.jax.org/vocab/mp_ontology/MP:0030435 
http://www.informatics.jax.org/vocab/mp_ontology/MP:0010743
http://purl.obolibrary.org/obo/HP_0001363
http://purl.obolibrary.org/obo/HP_0000270

* **Suggested term label:**
* coronal suture closure
* frontal suture closure
* lambdoid suture closure
* saggital suture closure
* metopic suture closure
* squamoparietal suture closure
* intermaxillary suture closure
* maxillary-premaxillary suture closure
* palatomaxillary suture closure
* zygomaticomaxillary suture closure
* frontonasal suture closure

* **Definition (free text)**
*The process of fusing together the edges of the [suture name] 
*Definitions for sutures without existing GO morphogenesis terms
**metopic suture: dense, fibrous connective tissue joint between the frontal bones from the sagittal suture to the root of the nose (source: https://radiopaedia.org/articles/metopic-suture | https://pubmed.ncbi.nlm.nih.gov/15882577/) 
** squamoparietal suture: dense, fibrous connective tissue joint between the squamous part of the temporal bone and the inferior aspect of the parietal bone (source: https://anatomyzone.com/articles/squamous-suture/ | https://www.worldcat.org/search?q=bn%3A0803606559) 
** intermaxillary suture: anterior median suture between the two maxillae of the upper jawbone, located between the inferior border of the nasal aperture and the central incisor teeth (source: https://www.imaios.com/en/e-Anatomy/Anatomical-Parts/Intermaxillary-suture)
** maxillary-premaxillary suture: the line of union of the two portions of the maxilla (source: https://www.worldcat.org/search?q=bn%3A0683400088)
** palatomaxillary suture: transverse suture across the back of the hard palate that separates the maxillary shelf of the palate from the horizontal plates of the palatine bones (source: https://www.imaios.com/en/e-Anatomy/Anatomical-Parts/Palatomaxillary-suture)  
** zygomaticomaxillary suture: the suture between the zygomatic process of the maxilla and the maxillary margin of the zygomatic bone (source: https://radiopaedia.org/articles/zygomaticomaxillary-suture)

* **Reference, in format PMID:#######**
Relevant PMIDs for mouse annotations:
21538817
7597092
9626498
10942429
15030764
22585574
15975938
20175913
21931569
24355923
12135917
12140681
12221714
14499350
15316116
17132737
17164422
20077479
23354439
10631169
11274405
12235118
12677005
15537544
15790973
16051666
16502419
18242159
19219044
20503384
20570969
25966306
28123344

* **Gene product name and ID to be annotated to this term**
Between premature and delayed suture closing there are around 150 mouse or human genes that may be able to be annotated to one or more of these terms
Fgfr1	MGI:95522
Bey	MGI:1276118
Fgfr2	MGI:95523
Axin2	MGI:1270862
Frem1	MGI:2670972
Tgfbr1	MGI:98728
Twist1	MGI:98872
Fgf9	MGI:104723
Dusp6	MGI:1914853
Gli3	MGI:95729
Hdac4	MGI:3036234
Erf	MGI:109637
Msx2	MGI:97169
Runx2	MGI:99829
Thrb	MGI:98743
Alkbh1	MGI:2384034
CASK	8573
CTSK	1513
Dlx5	MGI:101926
Fam20c	MGI:2136853
Fgf18	MGI:1277980
FGFR2	2263
Fgfrl1	MGI:2150920
FLNA	2316
Foxn3	MGI:1918625
GLI3	2737
Ibsp	MGI:96389
Kat6b	MGI:1858746
Lrp6	MGI:1298218
MED12	9968
Mettl5	MGI:1922672
Mmp14	MGI:101900
Sik3	MGI:2446296
Thra	MGI:98742
TWIST1	7291
Wls	MGI:1915401
AASS	10157
ADAMTS2	9509
ALDH18A1	5832
ALG9	79796
ALPL	249
ALX4	60529
AMER1	139285
ANKRD11	29123
ANTXR1	84168
ASPA	443
ATP6V0A2	23545
ATP7A	538
B3GAT3	26229
B3GLCT	145173
B4GALT7	11285
BMP4	652
BPNT2	54928
CCBE1	147372
CCDC88C	440193
CCNQ	92002
CDC45	8318
CDH11	1009
CEP57	9702
CEP120	153241
CHRNA1	1134
CHRND	1144
CHRNG	1146
CHST3	9469
CHST14	113189
COLEC11	78989
CREBBP	1387
CRTAP	10491
DMP1	1758
EFNB1	1947
ENPP1	5167
EP300	2033
ERF	2077
ESCO2	157570
FAM111A	63901
FAT4	79633
FGFR1	2260
FGFR3	2261
GLIS3	169792
GNPTAB	79158
GPC6	10082
GTF2E2	2961
H19-ICR	105259599
HSD17B4	3295
HUWE1	10075
IFT43	112752
IFT122	55764
IFT140	9742
IGF2	3481
IL6ST	3572
KAT6A	7994
KRAS	3845
KRT5	3852
KRT14	3861
L1CAM	3897
LEMD3	23592
LHX3	8022
LHX4	89884
MAF	4094
MASP1	5648
MEGF8	1954
MPDZ	8777
MSX2	4488
MYH3	4621
NAA10	8260
NFIX	4784
NLRP3	114548
NOS3	4846
OTX2	5015
P3H1	64175
P4HB	5034
PEX14	5195
PEX19	5824
POLA1	5422
POR	5447
POU1F1	5449
PPP1CB	5500
PROP1	5626
PTDSS1	9791
RAB23	51715
RECQL4	9401
ROR2	4920
RPS6KA3	6197
RPS19	6223
RUNX2	860
SCARF2	91179
SEC23A	10484
SEC24D	9871
SH3PXD2B	285590
SKI	6497
SLC12A6	9990
SLC35A2	7355
SLC39A8	64116
SMAD3	4088
SMC3	9126
STAT3	6774
STOX1	219736
TBC1D24	57465
TBCE	6905
TCF12	6938
TCIRG1	10312
TCTN2	79867
TGFBR1	7046
TGFBR2	7048
WDR19	57728
WDR35 	57539

* **Parent term(s)**
* coronal suture morphogenesis (GO:0060365)
* frontal suture morphogenesis (GO:0060364)
* lambdoid suture morphogenesis (GO:0060366)
* saggital suture morphogenesis (GO:0060367)
* cranial suture morphogenesis (GO:0060363)
* cranial suture morphogenesis (GO:0060363)
* facial suture morphogenesis (GO:0097096)
* facial suture morphogenesis (GO:0097096)
* facial suture morphogenesis (GO:0097096)
* facial suture morphogenesis (GO:0097096)
* frontonasal suture morphogenesis (GO:0097095

**Comment by pgaudet** [comment]
Hi @sbello 
We need one (or more) reference per term. Should we add all these references for ll these terms? 

Thanks, Pascale

**Comment by raymond91125** [comment]
@pgaudet Would these terms not be too granular for GO?

**Comment by pgaudet** [comment]
I am not familiar enough to judge - are the pathways for each of these characterized? If they are just phenotypic observations we would not create the terms. 

Is it possible in UPheno to use a uberon term in addition to a more general GO term?  

Thanks, Pascale

**Comment by sbello** [comment]
@pgaudet No, I've attached a spreadsheet that has the PMIDs for all of the mouse genes with the relevant annotations and terms shown
[suture_closure_annotations_references.txt](https://github.com/geneontology/go-ontology/files/8542974/suture_closure_annotations_references.txt)

@dosumis on the UPheno call expressed the opinion that we would be better served by having the GO process terms so that we could use the patterns and imports to place these annotations within the context of the GO tree. Using an Uberon term and the PATO term premature/delayed closure is our other option for a pattern but this loses all of the rich context we would get from using GO terms.

**Comment by cmungall** [comment]
I appreciate it's confusing for NTRs to be declared out of scope when in fact there are many existing terms of the same nature.

In this case we already have a bunch of inconsistently populated suture-specific terms

GO:0060363 ! cranial suture morphogenesis
GO:0060364 ! frontal suture morphogenesis
GO:0060365 ! coronal suture morphogenesis
GO:0060366 ! lambdoid suture morphogenesis
GO:0060367 ! sagittal suture morphogenesis
GO:0097094 ! craniofacial suture morphogenesis
GO:0097095 ! frontonasal suture morphogenesis
GO:0097096 ! facial suture morphogenesis
GO:0097097 ! nasal suture morphogenesis

On the one hand, the inconsistent population of these terms supports the position that this is not an ideal level of specificity in GO.

On the other hand it is very unclear for groups what is in or out of scope. If we had clear guidelines then we could make an extension ontology and use reasoning to merge it with GO for UPheno purposes. But we should not do this unless we have define clear scoping rules **and apply them consistently**.

Here is my own position on what we should do:

We should just grant these terms. It's consistent with levels of specificity right now in the ontology. I can add them. The additional maintenance burden will be a very small fraction of the existing burden of BP terms of varying specificity. This is a small lift and clearly important to a major community. (yes I am part of GO *and* Upheno). Surely in GO we want to support using the ontology to try and help provide mechanistic underpinnings of phenotypes like craniosynostosis (note: this is different from over-annotating phenotypes in GO).

To solve the inconsistent-specificity problem in GO we should just use do-not-annotate on overly specific terms. I think it would be fine to do this for the existing suture morphogenesis terms, there would be zero information loss and information would be at a consistent level.

We should also do some work on development over the next few years. We should be annotating processes like suture morphogenesis at the right *axis* of specificity. "morphogenesis" encompasses a lot of distinct subprocesses when it comes to suture formation, each with their own set of inductive signals, involving different substructures (osteoblasts, dura mater, ...).

**Comment by dosumis** [comment]
![image](https://user-images.githubusercontent.com/112839/166913694-b2afd74a-a3f0-4d68-a0cc-616c7e4c8af5.png)
https://www.frontiersin.org/articles/10.3389/fcell.2021.653579/full

From reading this I think
(a) the morphogenesis terms don't work.  I'm not sure whether they cover closure which seems to be described as the loss of sutures, but even if they do they are massively broader, covering process that shape and structure the sutures before closure. 
(b)  Closure appears to be an active process involving reorganisation of sharpies fibres and bone deposition.

So - I think at the very least GO should add  a term for 'cranial suture closure' with a logical relationship to uberon 'cranial suture'.  I'm agnostic as to whether GO should add all the subtypes for different cranial sutures (consistent with precedence set by morphogenesis terms) or now.  If GO does not add these, we can use a compound entity pattern to refer to them in phenotype ontologies, as long as GO sets the pattern for this in the parent class.  Maybe 'has_participant some 'cranial suture' would be sufficient?

**Comment by dosumis** [comment]
Update from uPheno meeting - the general term should be craniofacial suture closure

**Comment by nlharris** [comment]
Can this be closed? If not, can it be assigned to someone other than Chris?

**Comment by dosumis** [comment]
Would be great if this could be added soon.  Thanks!

**Comment by raymond91125** [comment]
Can someone advise a LD for suture closure? Thanks.

**Comment by raymond91125** [comment]
How about
'craniofacial suture morphogenesis' and ('results in fusion of' min 2 'head bone')

**Comment by dosumis** [comment]
* The cardinality restriction (min 2) is outside of EL so best avoided for reasoner compatibility/scaling.
* I think it would be more useful to have a pattern that referenced the sutures, rather than bones as this is how the morphogenesis terms are defined and following this pattern (in GO or in uPheno post comp) ensures correct classification of the suture closure by suture.
* It might be more correct to have closure be  part_of some 'craniofacial suture morphogenesis' rather than subClassOf (or, based on the screenshot text above, after closure it is not longer a suture, so could be a parallel class).  However, that would leave an open question around parent class, so maybe easiest to stick with this as superclass, especially if existing annotation for closure is to the morhpogenesis terms.
* I think we're just talking about closure 'cranial sutures' here (but more research may show justification for a broader term.)


Maybe: 

'cranial suture morphogenesis' and ('results in fusion of' some 'cranial suture')

Or - treating this as a parallel process:

'bone morphogenesis' and ('results in fusion of' some 'cranial suture')

**Comment by pgaudet** [comment]
This seems correct: 

> 'bone morphogenesis' and ('results in fusion of' some 'cranial suture')

but then we need to add the 'bone morphogenesis' parent. 

Also, discussing with @thomaspd - we're not clear whether this is morphogenesis of the suture or formation?

**Comment by raymond91125** [comment]
Is cranial suture closure sufficient? Several terms originally requested by @sbello are facial suture closure and one frontonasal suture.

**Comment by dosumis** [comment]
> Is cranial suture closure sufficient? Several terms originally requested by @sbello are facial suture closure and one frontonasal suture.

Sorry - my bad.  Use craniofacial.

**Comment by dosumis** [comment]
> Also, discussing with @thomaspd - we're not clear whether this is morphogenesis of the suture or formation?

See the Frontiers paper linked above.  Arguably it's the obliteration of the suture, but don't think we need to split hairs on this.

**Comment by raymond91125** [comment]
Thanks everyone for your help.
